src/app/components/find-journey/find-journey.component.html

```diff
@@ -1,6 +1,20 @@
 <div style="text-align:center">
   <h1>
+    <!--lets try to fetch the title from a son-->
     Welcome to {{ title }}!
   </h1>
 </div>
+<form class="example-form" (ngSubmit)="onSubmit()">
```

Comment:
for your information there is another way to create a form, we can discuss it later.

Reply:
discussed